.travis.yml

```diff
@@ -197,11 +213,96 @@ jobs:
                 - "$SMART_RUNNER_DIRECTORY"
             use:
               - node_modules_cache
-              - built_artefacts_cache
+              - built_libs_cache
               - smartrunner-cache-process
+              - built_demo_shell_cache
+
+        - stage: e2e Test
+          name: "Process Cloud : Form"
+          before_script:
+              - ./scripts/ci/job_hooks/before_e2e.sh
+              - ./scripts/ci/check-env/check-cs-env.sh || travis_terminate 1
+              - ./scripts/ci/check-env/check-ps-cloud-env.sh || travis_terminate 1
+          script: ./scripts/travis/e2e/process-services-cloud-e2e.sh
+          after_script: ./scripts/ci/job_hooks/after_e2e.sh
+          workspaces:
+            create:
+              name: smartrunner-cache-process-cloud
```

Comment:
I've just removed these in my PR, for smartrunner we don't use the Travis workspaces anymore for caching, only S3. You might get conflicts here

Reply:
my changes have been just merged, please don't forget to rebase